bnussman/Beep
# ✨ Improve sign up in app

## 📝 Description

- Updates the signup form for the app
- Now uses react hook form
- improved ui
- improved error handling

## 📷 Preview

<img height="680px" src="https://user-images.githubusercontent.com/6440455/169209864-beb4b910-0e5e-4070-b8af-ef413c6fb9cf.png" />

## 🧪 How to test

- Open app in browser or on Expo Go
- Check signup flow and ui
